charts/dapr/Chart.yaml

```diff
@@ -1,21 +1,21 @@
 apiVersion: v1
-appVersion: "0.7.1"
+appVersion: "0.8.0"
 description: A Helm chart for Dapr on Kubernetes
 name: dapr
-version: 0.4.1
+version: 0.4.2
```

Comment:
Do we have any rules defining when this should be a patch upgrade vs minor upgrade? I expected it to be a minor version bump

Reply:
Major changes / Breaking changes to the chart is a major version, minor can be non-breaking changes or fixes, patch fix is essentially changes to the metadata version only.